baselines/azure/azure_sub_import/main.tf

```diff
@@ -24,7 +24,6 @@ resource "turbot_resource" "subscription_resource" {
 }
 
 # Set the credentials for the subscription via Turbot policies
```

Comment:
I think it is useful to add the comment of what policy this is in comments. Look at the change that occurred.